README.md

```diff
@@ -199,126 +127,19 @@ Redoc uses the following [specification extensions](https://redocly.com/docs/api
 * [`x-extendedDiscriminator`](docs/redoc-vendor-extensions.md#x-extendedDiscriminator) - In Schemas, uses this to solve name-clash issues with the standard discriminator
```

Comment:
All bullets should be capitalized or none but having these two start with capitalized words when the others do not is inconsistent

Reply:
Fixed!

Reply:
Especially for a documentarian-targeted tool, this is a really good spot, thank you